Calibration/IsolatedParticles/src/CaloPropagateTrack.cc

```diff
@@ -507,6 +507,20 @@ namespace spr{
     return trkD;
   }
 
+  std::pair<bool,HcalDetId> propagateHCALBack(const reco::Track* track, const CaloGeometry* geo, const MagneticField* bField, bool debug) {
+    const CaloSubdetectorGeometry* gHB = geo->getSubdetectorGeometry(DetId::Hcal,HcalBarrel);
+    GlobalPoint  vertex (track->vx(), track->vy(), track->vz());
+    GlobalVector momentum (track->px(), track->py(), track->pz());
+    int charge (track->charge());
+    spr::propagatedTrack info = spr::propagateCalo(vertex, momentum, charge, bField, 549.3, 288.8, 1.392, debug);
```

Comment:
again magic numbers... from the function signature: `float zdist, float radius, float corner`. Is there a way to extract them from the geometry?